src/Microsoft.Azure.SignalR.Common/Exceptions/AzureSignalRForbiddenException.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.Azure.SignalR.Common;
+
+[Serializable]
+public class AzureSignalRForbiddenException : AzureSignalRException
+{
+    public const string IngressDenied = "Nginx denied the access, please check your Networking settings.";
```

Comment:
Ingress is also one component of Azure SignalR service from customer's perspective of view, let's not expose unnecessory terms to our customers.